src/integrationTest/groovy/wooga/gradle/unity/tasks/AddUPMPackagesTaskIntegrationSpec.groovy

```diff
@@ -54,21 +69,24 @@ class AddUPMPackagesTaskIntegrationSpec extends UnityTaskIntegrationSpec<AddUPMP
         def packages = ["com.unity.testtools.codecoverage": "1.1.0"]
         manifestFile << JsonOutput.toJson(["dependencies": packages])
 
-        and: "a setup AddUPMPackageTask"
-        appendToSubjectTask("""
-                buildTarget = "Android"
-                manifestPath = new File("build/test_project/Packages/custom/manifest.json")
+        and: "configuration of the AddUPMPackageTask"
+        addTask(taskName, AddUPMPackages, true, """
+                manifestPath = new File(${wrapValueBasedOnType(manifestPath, String)})
                 upmPackages.put("com.unity.package", "anyString")
             """.stripIndent()
         )
 
         when: "add UPM packages"
-        runTasksSuccessfully(subjectUnderTestName)
+        runTasksSuccessfully(taskName)
 
         then: "manifest file contains added packages"
         def dependencies = new JsonSlurper().parse(manifestFile)["dependencies"]
         dependencies["com.unity.testtools.codecoverage"] == "1.1.0"
         dependencies["com.unity.package"] == "anyString"
+
+        where:
+        taskName = "schonerKatzen"
```

Comment:
Ok why is it no longer using the `subjectUnderTestName`? Also I would refrain from non technical names for test values. Especialy if they show up in reports or logs. `schonerKatzen` in a log output etc for tests reads like some weird error.

Reply:
Because this is no longer an `UnityTask`. I did not want to add another `TaskIntegrationSpec`.

And sure.